CONTRIBUTING.md

```diff
@@ -94,7 +94,7 @@ Most operating systems provide all the needed tools (including Windows, Linux an
 
 - Mandatory:
 
-  - [Node.js](http://nodejs.org)
+  - [Node.js](http://nodejs.org) (^12.22.0 || ^14.17.0 || >=16.0.0, or >10 with using the `--ignore-engines` on Yarn)
```

Comment:
I am not convinced that this really explains the problem correctly. The engine specification mentioned here comes from the package @soryy708 tried to install: `@typescript-eslint/eslint-plugin`.

We have the same issue, but with different engine limitations coming from `@npmcli/map-workspaces`, one of our sub-dependencies (`"^12.13.0 || ^14.15.0 || >=16"`).

I would probably add a point 2.1 titled "Adding and updating dependencies", where we give a hint to the contributor, that they must use `yarn` and they might run into a `Found incompatible module` error

Reply:
Agree, thanks for the suggestion!